docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

```diff
@@ -13,24 +13,15 @@ keywords:
 
 # Publish or update a plugin
 
-You've just built your plugin; now you want to share it with the world.
+You've just built your plugin; now you want to share it with the world. In this guide, you'll learn how to publish a new plugin or update an existing one, and you'll learn about plugin deprecation.
 
-In this guide, you'll learn how to package and share your plugin with others.
-
-The best way to share your plugin with the world is to publish it in the [Grafana plugin catalog](https://grafana.com/plugins).
-By having your plugin published on Grafana.com, more users will be able to discover your plugin.
+The best way to share your plugin with the world is to publish it in the [Grafana plugin catalog](https://grafana.com/plugins). By having your plugin published on Grafana.com, more users will be able to discover your plugin.
 
 ## Before you begin
 
-When you build a plugin from source, a `dist` folder is created. This folder contains the production build or _plugin assets_ for your plugin.
-
-To package a plugin, refer to [Package a plugin](./package-a-plugin.md).
-
-### Follow our guidelines
-
-Get familiar with our plugin [publishing and signing criteria](./publishing-and-signing-criteria.md)
-
-### Do this for best results
+1. [Create a plugin](../get-started/get-started.mdx) - When you build a plugin from source, a `dist` folder is created. This folder contains the production build or _plugin assets_ for your plugin.
+1. [Follow our guidelines](https://grafana.com/legal/plugins/#plugin-publishing-and-signing-criteria) - Learn about the Grafana Labs criteria for publishing and signing plugins. 
+1. [Package a plugin](./package-a-plugin.md) - Build the plugin and get it ready to share in the form of a ZIP archive.
```

Comment:
yes, let's add that new point. I would hate to lose the examples link because we don't do a great job right now of publicizing them. I've moved it up to the previous bullet point list.